evaluationsjs.html

```diff
@@ -3,12 +3,13 @@
 	<head>
```

Comment:
Do you still need the other html files if index.html and index.js handles the entire view logic?